srml/authority-discovery/Cargo.toml

```diff
@@ -5,16 +5,16 @@ authors = ["Parity Technologies <admin@parity.io>"]
 edition = "2018"
 
 [dependencies]
+app-crypto = { package = "substrate-application-crypto", path = "../../core/application-crypto", default-features = false }
+babe-primitives = { package = "substrate-consensus-babe-primitives", path = "../../core/consensus/babe/primitives", default-features = false }
```

Comment:
This needs to go into `dev-dependencies`. You don't need it for the normal build.